.travis.yml

```diff
@@ -25,4 +27,4 @@ install:
     - composer config extra.symfony.allow-contrib true
 
 script:
-    - composer req "${PACKAGES}"
+    - composer req $(echo "$PACKAGES" | tr --delete '\n')
```

Comment:
Not sure this will work... The regex is just fetching one package...

Reply:
I'm not sure to understand what you mean by "The regex is just fetching one package". For this PR, `$PACKAGES` contains:

```
prooph/event-store-bus-bridge:^3.1
prooph/event-store-symfony-bundle:^0.4
prooph/pdo-event-store:^1.7
prooph/service-bus-symfony-bundle:^0.6
``` 

Which [will not be handled properly](https://travis-ci.org/symfony/recipes-contrib/jobs/357352746#L593) if using `composer req "${PACKAGES}"`.

The piped `tr` will allows to merge lines into a single line:
```
prooph/event-store-bus-bridge:^3.1 prooph/event-store-symfony-bundle:^0.4 prooph/pdo-event-store:^1.7 prooph/service-bus-symfony-bundle:^0.6
``` 

which will be correctly handled by `composer req`.

Reply:
Sorry for not being clear. `$PACKAGES` will never contain more that one package.

Reply:
Well, it does on this PR :thinking: